### PR TITLE
rough in infrastructure

### PR DESCRIPTION
**Pull request by mattldawson**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,98 @@
+cmake_minimum_required(VERSION 3.12)
+project(aerosol-radiation)
+enable_language(Fortran)
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
+set(CMAKE_CXX_STANDARD 17)
```
Comment (jeff-cohere) [inline CMakeLists.txt]: This is pretty aggressive! Are there working C++17 compilers now? :-)

Comment (mattldawson) [reply CMakeLists.txt]: Maybe 11? I copied some of this from another small repo I was trying out some things that used C++17

Comment (jeff-cohere) [reply CMakeLists.txt]: 11 or 14 should be okay. Let's start with 11 and see what's "missing."

File: driver/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+################################################################################
+# Mock host model drivers
+
+set( DRIVER_F90_SRC driver/compute_aero_optics.F90
```
Comment (jeff-cohere) [inline driver/CMakeLists.txt:4]: Any reason not to use `add_executable` here? We can always have them do a `make install` that splats the driver into a set location.

Comment (mattldawson) [reply driver/CMakeLists.txt:4]: I'm not sure I follow. Right now the drivers ends up in the build folder. Where were you proposing that they end up?

File: CMakeLists.txt (hunk 1 above)
Comment (mattldawson) [reply CMakeLists.txt]: done!

File: CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,98 @@
+cmake_minimum_required(VERSION 3.12)
+project(aerosol-radiation)
+enable_language(Fortran)
```
Comment (jeff-cohere) [inline CMakeLists.txt:3]: Maybe `enable_language(CXX)` here just for clarity.

**Review by jeff-cohere — APPROVED** [review]
Suggested a very minor change. Looks good!

File: CMakeLists.txt (hunk 3 above)
Comment (mattldawson) [reply CMakeLists.txt:3]: done!

**Pull request merged**